go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	k8s.io/code-generator v0.30.4
 	k8s.io/component-helpers v0.30.4
 	k8s.io/klog v1.0.0
+	k8s.io/klog/v2 v2.130.1
```

Comment:
This change was caused by running `go mod tidy`. 

Obviously both `v1` and `v2` are used in this project, which can be updated separately.

Reply:
I wonder why two versions are used

Reply:
https://github.com/kserve/kserve/blob/master/cmd/spec-gen/main.go#L26
https://github.com/kserve/kserve/blob/master/pkg/webhook/admission/pod/agent_injector.go#L26
https://github.com/kserve/kserve/blob/master/pkg/webhook/admission/pod/suite_test.go#L27

I don't want to touch them in this PR.